recipes/multiqc_sav/meta.yaml

```diff
@@ -5,11 +5,11 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/CenterForMedicalGeneticsGhent/MultiQC_SAV/archive/refs/tags/v{{ version }}.tar.gz
+  url: https://github.com/MultiQC/MultiQC_SAV/archive/refs/tags/v{{ version }}.tar.gz
   sha256: e0f743aa7b8c66ffa2eea59647564c378c06ea3122ff8d0b0e2363d18834efd2
 
 build:
-  number: 0
+  number: 1
```

Comment:
Do we need a new package? I can merge this in without a new package if that is ok for you.

Reply:
no, a new package isn't needed, the repo was moved, no further modifications were made.

Thanks
M

Reply:
Thanks!